gwcs/wcs.py

```diff
@@ -668,13 +668,24 @@ def to_fits_sip(self, bounding_box, max_error=None, max_rms=None, degree=None,
 
         Returns
         -------
-        fits_dict, max_error, rms_error
-
-        fits_dict is a dictionary where the keys are FITS keywords and the values
-        the corresonding FITS keyword values, to make it easy to merge with
-        an existing header.
+        FITS header with all SIP WCS keywords
 
         This assumes a tangent projection.
+
+        Error cases
+        -----------
```

Comment:
```suggestion
        Raises
        ------
```

Reply:
See https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html and search for **Raises**

Reply:
The thing is that you should try to build the documentation locally. Most likely you will get warnings/errors due "non-standard" section name (at least this is how it used to be...)

Reply:
Indeed, see https://travis-ci.org/spacetelescope/gwcs/jobs/655564310#L605-L607

Reply:
Done